### PR TITLE
Create E2E Test A

### PR DESCRIPTION
**Pull request by Devessier**

### CODE REVIEW

File: packages/client/e2e/test-a.spec.ts [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+    const focusTrap = creatorPage.locator('text="Click"').first();
+    await focusTrap.click();
+
+    await expect(creatorPage.locator('text="Home"').first()).toBeVisible();
+
+    const goToTracksSearch = creatorPage.locator('text="Search"');
+    await goToTracksSearch.click();
+
+    const trackQuery = 'BB Brunes';
+    await creatorPage.fill('css=[placeholder*="Search a track"]', trackQuery);
```
Comment (prastoin) [inline packages/client/e2e/test-a.spec.ts:209]: We might want to add `>> visibile=true`

File: packages/client/e2e/test-a.spec.ts [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+    const focusTrap = joinerPage.locator('text="Click"').first();
+    await focusTrap.click();
+
+    await joinerPage.click('text="Go to Music Track Vote"');
+
+    // Code to use infinite scroll
+    let matchingRoom: Locator | undefined;
+    let hasFoundRoom = false;
+
+    await joinerPage.mouse.move(
```
[the remark this answers is not in the document]
Comment (Devessier) [reply packages/client/e2e/test-a.spec.ts:314]: I do not thing that this is something related to E2E tests.
Here we scroll down until we find the room we are searching for, wherever other rooms are coming from.

File: packages/client/e2e/test-a.spec.ts (hunk 1 above)
Comment (Devessier) [reply packages/client/e2e/test-a.spec.ts:209]: I'll add it 👌

File: packages/client/e2e/test-a.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (prastoin) [reply packages/client/e2e/test-a.spec.ts:314]: Ok !

**Review by prastoin — DISMISSED** [review]
Lets goo
First of a long series !
Hype to see upcoming work :)
Ready to merge

**Review by prastoin — APPROVED** [review]
Fully secured now !

**Pull request merged**